src/pages/RouterCourse/routercourse.scss

```diff
@@ -0,0 +1,228 @@
+.page-router-course {
+  padding: 24px;
+
+  input,
+  button {
+    padding: .5em;
+    font-size: .9em;
+    margin-right: 10px;
+    border: none;
+    border-radius: 3px;
+    border: solid 1px #ccc;
+    outline: none;
+  }
+
+  button {
+    color: white;
+    background: #4188dc;
+    padding-left: 1em;
+    padding-right: 1em;
+    cursor: pointer;
+    border-color: rgba(0,0,0, .1);
+  }
+
+  .group {
+    &:after {
+      content: "";
+      display: table;
+      clear: both;
+    }
+  }
+
+  .course {
+    margin-bottom: 1em;
+
+    &-header {
+      margin-bottom: 40px;
+      padding-bottom: 20px;
+    }
+
+    &-nav {
+      li {
+        display: inline-block;
+
+        & + li {
+          margin-left: .6em;
+        }
+      }
+      a {
+        font-size: .85em;
+        color: #7d7d7d;
+        padding: .2em .8em;
+        border-radius: .25em;
+        background: #eaebeb;
+        transition: .2s;
+
+        &:hover,
+        &.active {
+          color: #fff;
+          background: #39add1;
+        }
+      }
+    }
+
+    &-img {
+      width: 70px;
+    }
+  }
+
+  .teachers {
+    display: flex;
+    flex-direction: column;
+
+    .teacher {
+      width: 100%;
+
+      &-img {
+        width: 100%;
+        border-radius: .3em;
+      }
+    }
+  }
+
+  .not-found {
+    text-align: center;
+  }
+
+  .icn {
+    &-logo {
+      background: #54606b;
+      color: #fff;
+
+      i {
+        font-size: 2.45em;
+        line-height: 0;
+        margin-top: 30px;
+      }
+    }
+
+    &-error {
+      font-size: 3.2em;
+      margin-bottom: .25em;
+    }
+  }
+
+  /* =================================
+    MEDIA QUERIES
+  ==================================== */
+
+  @media (min-width: 0) and (max-width: 768px) {
```

Comment:
Same here. We can remove `min-width: 0` from the logic.

Reply:
updated, removed min-width: 0